### ISSUE TITLE
Update native APIs

### ISSUE THREAD

**Issue by davidlehn** [digitalbazaar/rdf-canonize-native#6]
When the native code is split out to another module (see https://github.com/digitalbazaar/rdf-canonize/issues/5), also look at updating to N-API / node-addon-api.  Should help with some portability issues.

https://medium.com/the-node-js-collection/native-extensions-for-node-js-767e221b3d26

**Comment by mattcollier** [comment]
Start here: https://github.com/digitalbazaar/rdf-canonize-native/tree/napi

It works, but there seemed to be a significant performance penalty vs current implementation.  Hopefully it's just a matter of finding the offending code.

**Comment by davidlehn** [comment]
Yup, thanks for referencing that.  I was just trying the "transfer issue" feature since this was still in the js project.
